src/NServiceBus.Core/Serializers/XML/Serializer.cs

```diff
@@ -6,264 +6,100 @@
     using System.IO;
     using System.Linq;
     using System.Reflection;
-    using System.Text;
     using System.Xml;
     using System.Xml.Linq;
     using NServiceBus.Utils.Reflection;
 
+    class RawXmlTextWriter : XmlTextWriter
```

Comment:
Would this be better as a private nested class within `Serializer`? It is the only place it is used.

If there is a valid reason for it to remain internal then I would expect it to be in a separate file.